### PR TITLE
correct robot-environment configs and control scripts

### PR DESCRIPTION
**Pull request by dazza-codes**
- The robot environment is set to `stage` for the stage deployment systems and the matching environment files are in https://github.com/sul-dlss/shared_configs/tree/sdr-preservation-core_stage/config/environments
- The robot control script is consolidated into one control (ctl) script that accepts one argument to command what to do (start|stop|restart|status).
  - an issue to convert this to capistrano tasks, if possible, is in #60 but that will require more work that we have cycles for at the moment

### CODE REVIEW

File: bin/environment.rb [hunk 1]
```diff
@@ -18,9 +18,9 @@
   case `hostname -s`.chomp
     when "sdr-services", "sul-sdr-services"
       'production'
-    when "sdr-services-test"
-      'staging'
-    when "sul-sdr-services-dev"
+    when /sdr-services-test/
```
Comment (jcoyne) [inline bin/environment.rb]: Why are we switching to regexs here?

Comment (darrenleeweber) [inline bin/environment.rb]: There are two boxes with the same prefix, ending in `test` and `test2`.

Comment (jcoyne) [inline bin/environment.rb]: Can we be explicit with that then? 
```
when 'sdr-services-test', 'sdr-services-test2'
  ...
```

**Comment by darrenleeweber** [comment]
@jcoyne - removed regex pattern matches in favour of fixed string matches.

File: bin/sdr-preservation-core_ctl.sh [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+        export ROBOT_ENVIRONMENT='stage'
+        ;;
+
+    'sul-sdr-services-dev')
+        export ROBOT_ENVIRONMENT='integration'
+        ;;
+
+    *)
+        echo "WARNING: Failed to identify hostname"
+        export ROBOT_ENVIRONMENT='development'
```
Comment (jmartin-sul) [inline bin/sdr-preservation-core_ctl.sh]: should this be integration?  similar to line 26 above?  or maybe just an exit with an error, to be safe?

**Review by jmartin-sul — COMMENTED** [review]
one question, possibly just clarification for me, but otherwise, looks good.

File: bin/sdr-preservation-core_ctl.sh (hunk 2 above)
Comment (jmartin-sul) [inline bin/sdr-preservation-core_ctl.sh]: oh, maybe i see... is this a fall-through for the localhost case?  if so, maybe add that to the warning message?

**Comment by jcoyne** [comment]
Looks good to me. Deferring to @jmartin-sul on a final decision.

File: bin/sdr-preservation-core_ctl.sh (hunk 2 above)
Comment (darrenleeweber) [inline bin/sdr-preservation-core_ctl.sh]: Yes, it's a default for localhost development.  I'll modify this warning message.

**Comment by darrenleeweber** [comment]
@jmartin-sul & @jcoyne - thanks for the review questions, it's helped to improve this; it might be ready for prime time now.

**Pull request merged**